roles/umask/tasks/main.yml

```
@@ -1,4 +1,4 @@
 ---
 - name: "Configure system wide umask on {{ ansible_os_family }} to {{ umask_mode }}"
-  include_tasks: "{{ ansible_os_family|lower }}.yml"
-  when: umask_mode != false
+  ansible.builtin.include_tasks: "{{ ansible_os_family|lower }}.yml"
+  when: (umask_mode defined and umask_mode)
```

Is this correct? I thought is usually was
```suggestion
  when: (umask_mode is defined and umask_mode)
```

---

you're correct, the patch isn't ;-)

technically, `umask_mode` is always defined (it's set in `roles/umask/defaults/main.yml` to `false`), so you could omit the `is defined` part. or you could write

```
when:
  - umask_mode is defined
  - umask_mode
```